pipeline/misc/summary_stats_merger.ipynb

```diff
@@ -186,6 +181,8 @@
     "yml_path = pd.read_csv(yml_list,sep = \"\\t\").values.tolist()\n",
     "# Path to work directory where the yaml file locates\n",
     "#parameter: yml_path = path(\".\")\n",
+    "#if Ture, keep ambiguous alleles which can not be decided from flip or reverse, such as A/T or C/G. Otherwise, remove them.\n",
+    "parameter: keep_ambiguous = str\n",
```

Comment:
please change this to `parameter: keep_ambiguous = False` so we remove them by default, using a boolean variable.